src/storage/__tests__/mocks/session.mock.ts

```diff
@@ -0,0 +1,32 @@
+export const SESSION_MOCK = {
```

Comment:
Do we really want to put our mocks in a nested folder? Maybe we could put them higher in the tree to be able to reuse them in other tests, wdyt?